sdk/eventhub/azure-eventhubs/README.md

```diff
@@ -251,7 +251,7 @@ from azure.eventhub import EventData
 try:
     connection_str = '<< CONNECTION STRING FOR THE EVENT HUBS NAMESPACE >>'
     event_hub_path = '<< NAME OF THE EVENT HUB >>'
-    client = EventHubProducerClient.from_connection_string(connection_str, event_hub_path)
+    client = EventHubProducerClient.from_connection_string(connection_str, event_hub_path=event_hub_path)
 
     event_data_batch = await client.create_batch(max_size=10000)
     can_add = True
```

Comment:
All of these snippets have empty error handling:
```
except:
	raise
finally:
    pass
```

We should either show actual error handling, or just remove these altogether.

Reply:
remove all the try, except and finally.
and make all these samples can run directly.